README.md

```diff
@@ -80,11 +80,39 @@ For more information on packaging and distribution, refer to:
 - [Velopack Packaging Documentation](https://docs.velopack.io/category/packaging)
 - [Velopack Distribution Documentation](https://docs.velopack.io/category/distributing)
 
+## Using Github releases as your updates location
+There is an issue (https://github.com/velopack/velopack/issues/254) in the velopack repo about adding full api support for github releases. In the meantime 
+you can still use github releases with a few workarounds:
+### Download step (before packing the release) 
+Call `vpk download github --repoUrl "youRepoUrl"` before packing to include latest information from your github releases
+
+Then pack as normal.
+
+#### Uploading ( officially supported by vpk tool (https://docs.velopack.io/reference/cli/content/vpk-windows) )
+Use the 'github' flag in vpk command e.g 'vpk upload github'.
+
+Example python command:
+```
+ ret = subprocess.run(["vpk", "upload", "github", "--repoUrl", host_url, "--publish", "--releaseName", f"Appname {get_pubspec_yaml_version()}", "--tag", f"v{get_pubspec_yaml_version()}", "--token", your_github_token], check=True)
```

Comment:
Any reason for doing this example as a a python command instead of a regular shell prompt?

Reply:
No, makes more sense to just write out a regular shell command!